packages/react/src/components/ui/errors/ErrorPopup.tsx

```diff
@@ -10,7 +10,12 @@ type Props = {
 };
 
 const ErrorPopup = ({ customerSupportEmail }: Props) => {
-  return <UserError customerSupportEmail={customerSupportEmail} />;
+  return (
+    <UserError
+      customerSupportEmail={customerSupportEmail}
+      error={CorbadoError.name}
```

Comment:
This is not working right (we need to pass the error dynamically)?
I guess we need something like this here:

```

// by default, we show a generic error message with a customer support email
  const { t } = useTranslation('translation', { keyPrefix: 'errors.unexpectedError' });
  let errorString = customerSupportEmail ? t('withCustomerSupport', customerSupportEmail) : t('noCustomerSupport');

  // if we have more information though, we show a more detailed error (sometimes we show custom error messages at the top, e.g. for social logins)
  if (error.translatedMessage.length > 0) {
    errorString = error.translatedMessage;
  }

  return (
    <div className='cb-error-popup'>
      <div className='cb-error-popup-icon'>
        <ExclamationIcon className='cb-error-popup-icon' />
      </div>
      <div>
        <p className='cb-error-popup-text'>{errorString}</p>
      </div>
    </div>
  );

```